capsules/src/hmac.rs

```diff
@@ -227,9 +220,7 @@ impl<'a, H: digest::Digest<'a, T> + digest::HMACSha256, T: DigestType> digest::C
                         self.hmac.clear_data();
                         self.appid.clear();
 
-                        app.callback.map(|cb| {
-                            cb.schedule(usize::from(e.0), 0, 0);
-                        });
+                        app.callback.schedule(usize::from(e.0), 0, 0);
```

Comment:
Sorry, I missed this one on my first review, but this also needs to do the `ReturnCode` converesion